Source/CBLRestReplicator.m

```diff
@@ -82,7 +82,7 @@ @implementation CBLRestReplicator
     CBLCookieStorage* _cookieStorage;
 }
 
-@synthesize db=_db, settings=_settings, cookieStorage=_cookieStorage, serverCert=_serverCert;
+@synthesize db=_db, settings=_settings, serverCert=_serverCert, cookieStorage=_cookieStorage;
```

Comment:
I don't think we need `_cookieStorage` anymore. Can just access `_db.cookieStorage`.